src/workerd/tools/param-extractor.rs

```diff
@@ -0,0 +1,144 @@
+use std::{
+    fs::File,
+    io::{BufReader, BufWriter, Write},
+};
+
+use anyhow::Result;
```

Comment:
`anyhow` isn't included in the list of dependencies in `rust-deps/BUILD.bazel` under RTTI types because it was already in use -- when moving deps over here, that's something to keep in mind.